code/game/objects/items/ipc_overloaders.dm

```diff
@@ -0,0 +1,319 @@
+	. = ..()
+	to_chat(target, SPAN_GOOD("You feel good."))
+
+/obj/item/ipc_overloader/classic/midway_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	if(.)
+		to_chat(target, SPAN_NOTICE("You feel buzzed."))
+
+/obj/item/ipc_overloader/classic/finish_overloader_effect(var/mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_NOTICE("You sense the buzz is wearing off..."))
+
+
+// TRANQUIL
+/obj/item/ipc_overloader/tranquil
+	name = "GwokBuzz VanillaTranquil"
+	desc = "An overloader commonly marketed to IPC in high-intensity jobs by GwokBuzz, this drive is loaded with a program that reduces hostility within IPC and decreases risk analysis and projection."
+	icon_state = "vanilla"
+	var/list/static/calm_messages = list("You feel calm.", "You forget about your concerns.")
+
+/obj/item/ipc_overloader/tranquil/handle_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_GOOD(pick(calm_messages)))
+
+/obj/item/ipc_overloader/tranquil/midway_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	if(.)
+		to_chat(target, SPAN_GOOD(pick(calm_messages)))
+
+/obj/item/ipc_overloader/tranquil/finish_overloader_effect(var/mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_NOTICE("You feel the weight of the world return to your shoulders..."))
+
+
+// RAINBOW
+/obj/item/ipc_overloader/rainbow
+	name = "GwokBuzz Rainbow Essence"
+	desc = "An overloader commonly marketed to IPC in the service sector by GwokBuzz, this drive is loaded with a program that increases an IPC's perception that it has completed its current objectives or is more likely to complete its objectives."
+	icon_state = "rainbow"
+
+/obj/item/ipc_overloader/rainbow/handle_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_GOOD("You feel happy."))
+	target.druggy = 300
+
+/obj/item/ipc_overloader/rainbow/midway_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	if(.)
+		to_chat(target, SPAN_GOOD("You feel happy."))
+		target.druggy = 300
+
+/obj/item/ipc_overloader/rainbow/finish_overloader_effect(var/mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_NOTICE("The world seems a bit duller than it was before..."))
+	target.druggy = 0
+
+
+// SCREENSHAKER
+/obj/item/ipc_overloader/screenshaker
+	name = "GwokBuzz ScreenShaker"
+	desc = "An overloader commonly marketed to IPC in the service sector by GwokBuzz. This one causes an IPC's servos to pulse at random intervals, disrupting its balance."
+	icon_state = "screenshake"
+
+/obj/item/ipc_overloader/screenshaker/handle_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	to_chat(target, SPAN_NOTICE("You feel alert."))
+	target.dizziness = 0
+	target.make_dizzy(200)
+
+/obj/item/ipc_overloader/screenshaker/midway_overloader_effect(mob/living/carbon/human/target)
+	. = ..()
+	if(.)
+		to_chat(target, SPAN_NOTICE("You feel alert."))
+		target.dizziness = 0
```

Comment:
The make_dizzy proc already has a cap of 1000, do we need to set a cap for this drug specifically of 200? Is it like a lore/IC reason that plugging in two overloaders doesn't equal the effect of two overloaders?

Reply:
at the moment it's pretty hard to determine if it's a midway effect or multiple overloaders being used, and the idea of this one is that it's just a gentle screenshake, unlike the 1000 which goes ballistic, so i think for this it's fine with the 200 cap